include/nbl/video/IGPUVirtualTexture.h

```diff
@@ -90,21 +113,27 @@ class IGPUVirtualTexture final : public asset::IVirtualTexture<video::IGPUImageV
             params.samples = asset::IImage::ESCF_1_BIT;
             params.flags = static_cast<asset::IImage::E_CREATE_FLAGS>(0);
 
-            image = m_driver->createDeviceLocalGPUImageOnDedMem(std::move(params));
+            image = m_logicalDevice->createDeviceLocalGPUImageOnDedMem(std::move(params));
         }
 
     private:
         core::smart_refctd_ptr<IGPUImageView> createView_internal(IGPUImageView::SCreationParams&& _params) const override
         {
-            return m_driver->createGPUImageView(std::move(_params));
+            return m_logicalDevice->createGPUImageView(std::move(_params));
         }
 
-        IVideoDriver* m_driver = nullptr;
+        nbl::video::ILogicalDevice* m_logicalDevice;
     };
 
 public:
+
+    /*
+        The queue must have TRANSFER capability
+    */
```

Comment:
the queue's family, assert that property

Reply:
ok I've now added a refcounted backlink from createee to creator for ILogicalDevice and IPhysicalDevice respectively